src/us/kbase/workspace/test/database/mongo/S3BlobStoreIntegrationTest.java

```diff
@@ -46,6 +46,9 @@
 	// some reason.
 	
 	private static S3BlobStore s3bs;
+	// use to exercise the cert trusting code, although doesn't actually test against self
```

Comment:
Do you want to add code to test against a Minio with a self-signed cert, or do you want to test in one of our deploy environments instead?

Reply:
I'll test manually locally, and then we can test in CI. Running automated tests seems like a bit much